suite-native/module-send/src/screens/SendOutputsScreen.tsx

```diff
@@ -235,6 +236,25 @@ export const SendOutputsScreen = ({
         if (account) dispatch(updateFeeInfoThunk(account.symbol));
     }, [account, dispatch]);
 
+    useEffect(() => {
```

Comment:
This code is now duplicated across this and `SendFeesScreen,` right? Could you please create a hook that will be called on both screens? 

Like that, it will be easier to potentially modify it in a single place if needed.

Reply:
I was thinking about it, but his felt more explanatory. But if you are also thinking about it, I'll better do the hook 👍

Reply:
1c7ef791c5